src/SprykerEco/Yves/Payolution/Form/InstallmentSubForm.php

```diff
@@ -19,14 +19,14 @@
 
 class InstallmentSubForm extends AbstractPayolutionSubForm
 {
-    const PAYMENT_PROVIDER = PayolutionConfig::PROVIDER_NAME;
-    const PAYMENT_METHOD = 'installment';
-    const FIELD_INSTALLMENT_PAYMENT_DETAIL_INDEX = 'installment_payment_detail_index';
-    const FIELD_BANK_ACCOUNT_HOLDER = 'bank_account_holder';
-    const FIELD_BANK_ACCOUNT_IBAN = 'bank_account_iban';
-    const FIELD_BANK_ACCOUNT_BIC = 'bank_account_bic';
+    public const PAYMENT_PROVIDER = PayolutionConfig::PROVIDER_NAME;
+    public const PAYMENT_METHOD = 'installment';
+    public const FIELD_INSTALLMENT_PAYMENT_DETAIL_INDEX = 'installment_payment_detail_index';
+    public const FIELD_BANK_ACCOUNT_HOLDER = 'bank_account_holder';
+    public const FIELD_BANK_ACCOUNT_IBAN = 'bank_account_iban';
+    public const FIELD_BANK_ACCOUNT_BIC = 'bank_account_bic';
```

Comment:
I suppose not all of these constants should be public as their usage is located in this class only. Please check which can be marked as protected.

Reply:
And also please check constants are used through `static` key word not `self`.